Describe megacounties, combined sensor, Google drop, Quidel counties (#105)
cmu-delphi/delphi-epidata: docs/api/covidcast_signals.md

[COVID-19 GitHub repository](https://github.com/CSSEGISandData/COVID-19) without 
filtering, smoothing, or changes.

### `combination`

> This source included signals created as statistical combinations of the other

Reviewer: ```suggestion
This source provides signals which are statistical combinations of the other
```

> This source included signals created as statistical combinations of the other
> sources; it is not a primary data source, but instead is calculated using
> signals from the other sources above.

Reviewer: ```suggestion
This source provides signals which are statistical combinations of the other
sources above. It is not a primary data source.
```
